third_party/blink/renderer/modules/mediasource/media_source.cc

```diff
@@ -492,13 +498,48 @@ bool MediaSource::isTypeSupported(ExecutionContext* context,
   bool result = IsTypeSupportedInternal(
       context, type, true /* Require fully specified mime and codecs */);
   DVLOG(2) << __func__ << "(" << type << ") -> " << (result ? "true" : "false");
+#if BUILDFLAG(IS_COBALT)
```

Comment:
nit: do we need to check IS_COBALT and USE_STARBOARD_MEDIA here?

Reply:
Done

Reply:
Done